[14.0][FIX] sign_oca: Small fixes in configure mode

BernatPForgeFlow opened the pull request:
- Load 'required' field in configure dialog
- Show 'required' fields in red in configure mode
- Load 'role_id' field in configure dialog

BernatPForgeFlow committed:
[FIX] sign_oca: Small fixes in configure mode

- Load 'required' field in configure dialog
- Show 'required' fields in red in configure mode
- Load 'role_id' field in configure dialog

etobella reviewed (approved):
Can you squash it in a single commit?

BernatPForgeFlow commented:
> Can you squash it in a single commit?

@etobella I squashed the commits and I added another fix to correctly load the role. There was a mix between 'role' and 'role_id' that made it to not work. Can you review?

pedrobaeza reviewed (approved):
/ocabot merge minor
